src/scripts/store.js

```diff
@@ -0,0 +1,325 @@
+  //   visited: true,
+  // },
+  // sofia123: {
+  //   id: "sofia123",
+  //   city: "Sofia",
+  //   country: "Bulgaria",
+  //   photo: require("../img/sofia.jpg"),
+  //   visited: true,
+  // },
+  // bansko123: {
+  //   id: "bansko123",
+  //   city: "Bansko",
+  //   country: "Slovakia",
+  //   photo: require("../img/bansko1.jpg"),
+  //   visited: true,
+  // },
+  // varna123: {
+  //   id: "varna123",
+  //   city: "Varna",
+  //   country: "Bulgaria",
+  //   photo: require("../img/varna.jpg"),
+  //   visited: true,
+  // },
+  // solun123: {
+  //   id: "solun123",
+  //   city: "Thessaloniki",
+  //   country: "Greece",
+  //   photo: require("../img/thessaloniki.jpg"),
+  //   visited: true,
+  // },
+  // istanbul123: {
+  //   id: "istanbul123",
+  //   city: "Istanbul",
+  //   country: "Turkey",
+  //   photo: require("../img/istanbul.jpg"),
+  //   visited: true,
+  // },
+  // //Wishlist
+  // paris123: {
+  //   id: "paris123",
+  //   city: "Paris",
+  //   country: "France",
+  //   photo: require("../img/paris.jpg"),
+  //   visited: false,
+  // },
+  // santorini123: {
+  //   id: "santorini123",
+  //   city: "Santorini",
+  //   country: "Greece",
+  //   photo: require("../img/santorini.jpg"),
+  //   visited: false,
+  // },
+  // newyork123: {
+  //   id: "newyork123",
+  //   city: "New York",
+  //   country: "U.S.",
+  //   photo: require("../img/newyork.jpg"),
+  //   visited: false,
+  // },
+  // marrakech: {
+  //   id: "marrakech",
+  //   city: "Marrakech",
+  //   country: "Morocco",
+  //   photo: require("../img/marrakech.jpg"),
+  //   visited: false,
+  // },
+  // amsterdam123: {
+  //   id: "amsterdam123",
+  //   city: "Amsterdam",
+  //   country: "Netherlands",
+  //   photo: require("../img/Amsterdam.png"),
+  //   visited: false,
+  // },
+  // berlin123: {
+  //   id: "berlin123",
+  //   city: "Berlin",
+  //   country: "Germany",
+  //   photo: require("../img/berlin.jpg"),
+  //   visited: false,
+  // },
+  // lisbon123: {
+  //   id: "lisbon123",
+  //   city: "Lisbon",
+  //   country: "Porugal",
+  //   photo: require("../img/lisbon.jpg"),
+  //   visited: false,
+  // },
+  // tokyo123: {
+  //   id: "tokyo123",
+  //   city: "Tokyo",
+  //   country: "Japan",
+  //   photo: require("../img/tokyo.jpg"),
+  //   visited: false,
+  // },
+  // sienna123: {
+  //   id: "sienna123",
+  //   city: "Sienna",
+  //   country: "Italy",
+  //   photo: require("../img/sienna.jpg"),
+  //   visited: false,
+  // },
+  // dublin123: {
+  //   id: "dublin123",
+  //   city: "Dublin",
+  //   country: "Ireland",
+  //   photo: require("../img/dublin.jpg"),
+  //   visited: false,
+  // },
+  // petersburg123: {
+  //   id: "petersburg123",
+  //   city: "Saint petersburg",
+  //   country: "Russia",
+  //   photo: require("../img/saintpetersburg.jpg"),
+  //   visited: false,
+  // },
+};
+
+export function toggleButtonVisited(item) {
+  item.visited = true;
```

Comment:
@majac91 this PR is almost ready! The only thing missing now is to make this `visited` dynamic (true or false)!

You can toggle the status dynamically like this:

```js
const visited = item.visited
item.visited = !visited;
```